test/FunctionalTests/Microsoft.OData.Edm.Tests/Csdl/Parsing/SchemaJsonParserTests.cs

```diff
@@ -286,13 +286,15 @@ public void ParseCsdlTypeDefinitionWithMembersWorksAsExpected()
         #endregion
 
         #region Term
+
         [Fact]
-        public void ParseCsdlTermWithMembersWorksAsExpected()
+        public void ParseCsdlTermWithBaseTermAndMembersWorksAsExpected()

```

Comment:
>ParseCsdlTermWithBaseTermAndMembersWorksAsExpected [](start = 20, length = 50)

Can you also add some XML test variants of reading/writing terms with base values?

Reply:
There's no related test cases to test part of an XML Element.

I added a whole write test cases to cover it.

---
In reply to: [545398235](https://github.com/OData/odata.net/pull/1962#discussion_r545398235) [](ancestors = 545398235)